platform/platform-impl/src/com/intellij/ide/actions/AboutPopup.java

```diff
@@ -410,7 +419,27 @@ protected JBPoint getCopyIconCoord() {
     }
 
     public String getText() {
-      return myInfo.toString() + SystemInfo.getOsNameAndVersion();
+      Project project = CommonDataKeys.PROJECT.getData(DataManager.getInstance().getDataContextFromFocus().getResultSync());
```

Comment:
We don't think that mixing project specific information with Application info is a good idea. Also, it could break user's privacy. However, EP for providing some project independent information like OS version, etc is a viable idea. I'll take a look what we can improve here